lib/collector/remote-method.js

```diff
@@ -76,13 +76,20 @@ RemoteMethod.prototype.invoke = function invoke(payload, nrHeaders, callback) {
     nrHeaders = null
   }
 
-  if (!payload) payload = []
+  if (!payload) {
+    payload = []
+  }
   logger.trace('Invoking remote method %s', this.name)
 
-  this.serialize(payload, function onSerialize(err, serialized) {
-    if (err) return callback(err)
-    this._post(serialized, nrHeaders, callback)
-  }.bind(this))
+  this.serialize(
+    payload,
+    function onSerialize(err, serialized) {
+      if (err) {
+        return callback(err)
+      }
+      this._post(serialized, nrHeaders, callback)
+    }.bind(this)
+  )
```

Comment:
I had to double-take this refactor to realize it was binding the 'onSerialize' function. +1 prettier.

Reply:
yea, this is a good example of prettier doing what it says it'll do